### ISSUE TITLE
No vehicle found 

### ISSUE THREAD

**Issue by pdo805** [itsMeDavidV/Watchla-Support#217]
hi there, 
I’m on latest ios app version as of 1/14/24. I deleted app, reinstalled and tried out and signed back in many times…still no luck

Please advise on how to fix this. Thanks

**Comment by rfelix2121** [comment]
Same here. Even though I've authenticated with Tesla, it says I have no vehicles no matter how much I sign in/out and reinstall/uninstall.

**Comment by matkohlk** [comment]
Same here. Please advise.

**Comment by hosa85** [comment]
Same here, quite frustrating as the app was still working until I signed out as the app told me too

maybe this should be removed if the API / App update is bricked

**Comment by getnorm** [comment]
Same issue.
Re-installed new version of app and now cannot detect vehicle.

**Comment by m-kyr** [comment]
Same!!!

**Comment by itsMeDavidV** [comment]
Hi all, this issue should be fixed in v3.5.2. If this issue is not quite fixed for you, please reach out with your support code here: https://watchla.app/contact with a short description of what exactly is happening leading up to this issue. Thanks!

**Comment by m-kyr** [comment]
Just sent the feedback, hope it helps.Sent from my iPhoneOn Jan 18, 2024, at 3:14 PM, David Villegas ***@***.***> wrote:﻿
Hi all, this issue should be fixed in v3.5.2. If this issue is not quite fixed for you, please reach out with your support code here: https://watchla.app/contact with a short description of what exactly is happening leading up to this issue. Thanks!

**Comment by matkohlk** [comment]
Unfortunately the issue not fixed. I have this latest v3.5.2, but still same issue. 


Sent from Yahoo Mail for iPhone


On Thursday, January 18, 2024, 4:49 PM, m-kyr ***@***.***> wrote:



Just sent the feedback, hope it helps.Sent from my iPhoneOn Jan 18, 2024, at 3:14 PM, David Villegas ***@***.***> wrote:﻿
Hi all, this issue should be fixed in v3.5.2. If this issue is not quite fixed for you, please reach out with your support code here: https://watchla.app/contact with a short description of what exactly is happening leading up to this issue. Thanks!

**Comment by getnorm** [comment]
> Just sent the feedback, hope it helps.Sent from my iPhoneOn Jan 18, 2024, at 3:14 PM, David Villegas ***@***.***> wrote:﻿ Hi all, this issue should be fixed in v3.5.2. If this issue is not quite fixed for you, please reach out with your support code here: https://watchla.app/contact with a short description of what exactly is happening leading up to this issue. Thanks! —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

Am running v3.5.2 and App still is not detecting vehicle.

**Comment by pdo805** [comment]
@getnorm, I've updated to latest IOS version as well and still having same issue.

I've sent watchla.app/contact this issue along with support code for a closer look.  Hope they found root cause of this issue

**Comment by mauricio2099** [comment]
Any update on this? I am on the latest version as of today 3.5.3 and I am still seeing the No vehicle found issue.
Does the app need more permissions to work apart from Commands?

**Comment by CassandraMarie** [comment]
Hi @pdo805, @rfelix2121, @mauricio2099, @m-kyr, @hosa85, @getnorm, and @matkohlk, we apologize for the late reply. Are you still experiencing this issue?

**Comment by m-kyr** [comment]
Nope, works for me now, I had to give all the permissions again. I know this is not the right place, but watch app needs to be open to unlock the car, it doesn't work in the background.

**Comment by CassandraMarie** [comment]
Hi @m-kyr, Can you send over your support code via [https://watchla.app/contact/](https://watchla.app/contact/) so we can investigate your case?

Support codes can be found by going to Preferences > Support/ Feedback > Support code
Please be sure to link to this issue when you reach out.
